src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
```

Comment:
These are the other properties we define by default in shipping packages:

Description
PackageTags
GenerateDocumentationFile= true

Example:
https://github.com/aspnet/IISIntegration/blob/db84780e9369112d578c2733d2459a5972d15227/src/Microsoft.AspNetCore.Server.IISIntegration/Microsoft.AspNetCore.Server.IISIntegration.csproj#L4-L8